### DISCUSSION TITLE
What's the difference between the nonfree & free ISOs?

### DISCUSSION THREAD

**Discussion by taoeffect** [Start9Labs/start-os#2407] [Q&A] [answered]
Here are several ISOs in the releases: https://github.com/Start9Labs/start-os/releases/tag/v0.3.4.4-hotfix

Some of them are labeled "nonfree" while others are not. What's the difference between these and are there any tradeoffs?

**Comment by taoeffect** [comment]
I may have partially answered my question as I found out that I can't use the free ISO from Parallels on macOS. I'm guessing there might be similar hardware issues for other environments.

**Comment by Satoshiroll** [comment, marked as answer]
The non-free images contain proprietary device drivers, that are needed by most DIY hardware.

https://docs.start9.com/latest/guides/flashing/flashing-x86#getting-startos

**Reply by Jplourde00** [reply]
What about a bee link s13

**Reply by Jplourde00** [reply]
Im having real trouble booting the nonfree. Disabled secure boot
